automated_testing/test/tests/submissions-details-test.ts

```diff
@@ -0,0 +1,59 @@
+import AuthPage from '../pageobjects/auth-page';
+import SubmissionsDetailsPage from '../pageobjects/submissions-details-page';
+
+describe('Testing SubmissionsDetails Page', () => {
+    const validUserCredentials = {
+        username: 'testuser123',
+        password: '123456',
+    };
+
+    const checker = { notEmpty: (itemsList) => itemsList.length > 0 };
+
+    const loginWithCredentials = async (credentials) => {
+        const { username, password } = credentials;
+        await AuthPage.performLogIn(username, password);
+        return validUserCredentials;
+    };
+
+    const loginInWithValidCredentials = () => loginWithCredentials(validUserCredentials);
+
+    before(async () => {
+        await AuthPage.open();
+        await loginInWithValidCredentials();
+        await SubmissionsDetailsPage.open();
+    });
+
+    after(async () => {
+        await AuthPage.performLogOut();
+    });
+
+    it('Expect left navigation to be there', async () => {
+        await browser.waitUntil(
+            () => browser.execute(() => document.readyState === 'complete'),
```

Comment:
we can extract this into a function and define the failure message

Reply:
This was witten by you and it was only fo sanity check and I will delete it when the time comes. Also we discussed yesterday that the focus of this PR is only the class change and the tests part are in it by mistake.